.github/workflows/integration.yaml

```diff
@@ -21,15 +21,21 @@ jobs:
       matrix: ${{ steps.supported-version.outputs.matrix }}
     steps:
       - uses: actions/checkout@v2
-      - uses: graycoreio/github-actions-magento2/supported-version@main
+      - uses: graycoreio/github-actions-magento2/supported-version@fixup_mysql_versions_for_tests
         id: supported-version
+        with:
```

Comment:
Let's pull this back now that we know that it works. Is there a minimal test that we can include that proves that we've covered this issue? I'd like to ignore Magento core tests if possible. It's not our responsibility to run them.

Reply:
Added a `UniqueExecutorTest` and removed Magento core tests.